src/core/thread/dua_manager.cpp

```diff
@@ -508,7 +508,8 @@ void DuaManager::PerformNextRegistration(void)
     mIsDuaPending   = true;
     mRegisteringDua = dua;
 
-    // TODO: (DUA) need update when CSL is enabled.
+    // Regardless of csl enabled or not, sleepy end device should always rely on
+    // fast data poll to fetch the response timely when the parent is of version 1.1.
```

Comment:
This comment doesn't seem clear to me. CSL cannot be used when parent "is of version 1.1". Can we make this more clear?

Reply:
Updated. Please take another review.

Reply:
Much more clear. Thanks!